### DISCUSSION TITLE
pore occupancy.

### DISCUSSION THREAD

**Discussion by kizzle4shaw** [PMEAL/OpenPNM#2710] [Q&A]
Is there a way to quantify pore occupancy of an individual phase within the pore assuming it air and water like a probability distribution function ?

**Comment by jgostick** [comment]
This question is too vague.  Can you re-ask with a bit more context?

**Reply by kizzle4shaw** [reply]
What I meant is from simulation output, how do i get saturation vs time? how to see saturation change in the network.
